.github/workflows/release-please.yml

```diff
@@ -18,6 +18,8 @@ jobs:
           package-name: 'skyux'
           pull-request-title-pattern: 'chore: release ${version}'
           labels: 'autorelease ${{ github.ref_name }}: pending'
+          last-release-sha: 'bc28ca0df0183146f92482c396409d0369ae4532'
```

Comment:
This is the last SHA of `6.x.x` release, so that we can pull in all commits in the beta release.